### PR TITLE
Added workplan changed event to have the possibility to react on changes

### PR DESCRIPTION
**Pull request by slawor**
It is necessary to have an event about a changed workplan to be able to react on this change.

### CODE REVIEW

**Comment by Toxantron** [comment]
Since we have to extend the workplan API anyway. Should we include a `SourceId` for workplans. This could be helpful for future changes when workplans are cloned, versioned, ...
@dbeuchler ?

File: src/Moryx/Workflows/API/IWorkplanManagement.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Moryx.Workflows
+{
+    public interface IWorkplanManagement : IWorkplans
```
Comment (Toxantron) [inline src/Moryx/Workflows/API/IWorkplanManagement.cs:5]: Can you disable the Deprecated warning for this usage?

**Review by Toxantron — CHANGES_REQUESTED** [review]

**Comment by dbeuchler** [comment]
> Since we have to extend the workplan API anyway. Should we include a `SourceId` for workplans. This could be helpful for future changes when workplans are cloned, versioned, ...
> @dbeuchler ?

yes... source id can be helpfull

**Comment by slawor** [comment]
> Since we have to extend the workplan API anyway. Should we include a `SourceId` for workplans. This could be helpful for future changes when workplans are cloned, versioned, ...
> @dbeuchler ?

Can we add it if we need it in the future? I would like to address only one problem

**Review by Toxantron — APPROVED** [review]

**Comment by Toxantron** [comment]
Since the requirement has disappeared, could we do this against future without obsolotes?

**Pull request closed without merging**